src/atsu/src/com/HoppieConnector.ts

```diff
@@ -128,6 +131,113 @@ export class HoppieConnector {
         return AtsuStatusCodes.NoHoppieConnection;
     }
 
+    private static levenshteinDistance(template: string, message: string, content: CpdlcMessageContent[]): number {
```

Comment:
Seems like a good candidate for A32NX_Util or similar rather than in the hoppie connector.

Reply:
The problem is that A32NX_Util is something in the JS-code and we try to avoid code-sharing between or TS-modules and the released JS-code. For the moment is Hoppie the only one who needs string matching.

POSCON will directly use the ICAO-CPDLC-IDs for the network protocol.